common/lib/xmodule/xmodule/assets/vertical/public/js/vertical_student_view.js

```diff
@@ -13,4 +13,25 @@ window.VerticalStudentView = function(runtime, element) {
             apiUrl: $bookmarkButtonElement.data('bookmarksApiUrl')
         });
     });
+    console.log("jcd VerticalStudentView loaded");
+    $(element).find('.vert').each(function(idx, completable) {
+        console.log(completable.dataset);
+        if (completable.dataset.completable != "True") {
```

Comment:
Nit: I'm not a fan of this string "True" comparison. How about putting the `data-completable` attribute (with no value) on the element if and only if it should be marked as completed on view (or perhaps call it `data-complete-on-view`?), and then change this check here to use `hasAttribute`?

i.e. https://stackoverflow.com/questions/16864999/can-i-use-html5-data-attributes-as-boolean-attributes

Reply:
Sounds good.  I don't love "stringly" typed data either.  I'll do it that way.

I suppose another option would be to make the values "JSON like" (so `data-bool='true' data-null='null' data-num='1' data-string='"string"' data-object='{"tweedle": "dee"}'`), but you'd need to make that a site-wide policy to be worth doing anyway.  (just musing out loud).